package.json

```diff
@@ -51,7 +51,8 @@
     "carbon-components": "^10.6.0",
     "carbon-components-react": "^7.7.0",
     "carbon-icons": "^7.0.7",
-    "cross-env": "^7.0.2",
+    "concurrently": "^5.2.0",
+    "cross-env": "^5.1.4",
```

Comment:
Why did we revert to an older version of `cross-env`?  Does it have to with `concurrently`?

Reply:
That's a good question. I'm guessing that we upped `cross-env` versions while this pr was in the wings, then maybe I crossed out the wrong one during a conflict resolution. I'll try changing it back to 7 and see if anything breaks...

Reply:
Everything looked alright to me